src/utils.js

```diff
@@ -183,6 +184,8 @@ export const setHttpWrapped = () => (process.env[HTTP_WRAPPED_FLAG] = 'TRUE');
 
 export const isStoreLogs = () => validateEnvVar(STORE_LOGS_FLAG);
 
+export const isReuseHttpConnection = () => validateEnvVar(REUSE_CONNECTION);
```

Comment:
Don't we want the default to be `true`?

Reply:
In the feature yes, for now, we want to test this internally first (on backend-monitoring) and then release GA